wave/src/User.php

```diff
@@ -85,8 +99,13 @@ public function subscriber(){
         return false;
     }
 
-    public function subscription(){
-        return $this->hasOne(PaddleSubscription::class);
+    public function subscription($name = 'default')
+    {
+        if (env('CASHIER_VENDOR') == 'stripe') {
+            return $this->subscriptions->where('name', $name)->first();
+        } else {
+            return $this->hasOne(PaddleSubscription::class);
+        }
```

Comment:
```suggestion
        }

        return $this->hasOne(PaddleSubscription::class);
```

The `else` block isn't needed here since we `return` early from the `if` statement if it's true.